go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -113,17 +113,15 @@ func (oc *Controller) syncEgressFirewall(egressFirwalls []interface{}) {
 	// update the direction of each egressFirewallACL if needed
 	if len(egressFirewallACLs) != 0 {
 		opModels := []libovsdbops.OperationModel{}
-		for i, egressFirewallACL := range egressFirewallACLs {
-			aclName := egressFirewallACLs[i].Name
+		for i := range egressFirewallACLs {
+			egressFirewallACL := egressFirewallACLs[i]
```

Comment:
nit: would have been cleaner to just do:

```
for _, egressFirewallACL := range egressFirewallACLs {
```

Reply:
@trozet We found that this didn't work when using `egressFirewallACL.<FIELD>` within an anonymous function, It was a really strange behavior but this was the workaround

Reply:
This is because in a for/range loop, golang uses the same memory for the variables, so we can no store that memory address for further use.